test/exercises_test.dart

```diff
@@ -76,7 +78,7 @@ Running tests for: $packageName
       ["sed", "-i", "-e", "s/\\bskip:\\s*true\\b/skip: false/g", "test/${packageName}_test.dart"],
 
       /// Pull dependencies
-      ["pub", "get"],
+      ["pub", "upgrade"],
```

Comment:
Why `upgrade` instead of `get`? Couldn't that cause unforeseen issues during CI?

Reply:
Because contributors may find themselves switching between Dart 1.0 and Dart 2.0 `pub get` doesn't resolve the problems such a switch would usually have.

By running `pub upgrade` it forces pub to change the packages used to fit the SDK requirements.

Otherwise, contributors would have to go to **each** and *every* exercise folder and run `pub upgrade`.

An alternative to the current change, would be we create a script that just does `pub upgrade` in each exercise folder, but I'm not sure I like that.

I'm open to suggestions.

Reply:
@jvarness @devkabiir I reverted this change, just know you'd have to remove all the .dart_tool/, .pub/, .packages, and pubspec.lock files for all the exercises and the root when you switch between Dart 2 and 1.